main.go

```diff
@@ -108,6 +109,9 @@ func watchConfig(fileName string, mapper *metricMapper) {
 func main() {
 	flag.Parse()
 
+	if *addSuffix {
+		log.Println("Warning: Using -statsd.add-suffix is discouraged. Better explicitly name metrics in config according to best pratices.")
```

Comment:
Instead of "Better..." I'd go with "We recommend explicitly naming metrics appropriately in the mapping configuration."